sdk/core/azure-core/test/ut/telemetry_policy_test.cpp

```diff
@@ -100,15 +100,15 @@ TEST(TelemetryPolicy, UserAgentCppVer)
     EXPECT_EQ(cppversion.str(), "TEST:201402");
   }
   {
-    const std::string ua = Http::_internal::UserAgentGenerator::GenerateUserAgent(
+    const std::string ua = Http::_internal::UserAgentHelper::BuildUserAgent(
```

Comment:
Is this a diff issue?

Reply:
I did not check in some changes, please see the latest iteration.

Reply:
I saw the same thing, refresh ;)